src/main/java/com/neverpile/urlcrypto/springsecurity/GeneratePreSignedUrlInterceptor.java

```diff
@@ -49,20 +52,41 @@ public boolean preHandle(final HttpServletRequest request, final HttpServletResp
             private static final long serialVersionUID = 1L;
           };
         } else {
-          Duration expiryTime = parse(requestedExpiryTime);
-
-          // limit validity to configured maximum
-          if (null != config.getMaxPreSignedValidity() && expiryTime.compareTo(config.getMaxPreSignedValidity()) > 0)
-            expiryTime = config.getMaxPreSignedValidity();
-
-          String url = crypto.generatePreSignedUrl(expiryTime, request.getRequestURL().toString());
-
-          response.setContentType("text/uri-list");
-          response.getWriter().write(url + "\r\n");
-          return false;
+          return generatePreSignedUrl(request, response, requestedExpiryTime);
+        }
+      } else if (handler instanceof ResourceHttpRequestHandler) {
+        if (isEnabledStaticPath(request)) {
+          throw new AuthenticationException("Pre Sign URLs(PSU) not enabled for " + request.getRequestURL().toString()) {
```

Comment:
Hier könnte man in der ExceptionMessage noch ergänzen: "Please configure neverpile.url-crypto.psuEnabledPathPatterns accordingly."  (oder so ähnlich).